src/playbacks/html5_video/html5_video.js

```diff
@@ -383,7 +383,15 @@ export default class HTML5Video extends Playback {
   }
 
   _onError() {
-    this.trigger(Events.PLAYBACK_ERROR, this.el.error, this.name)
+    const { code, message } = this.el.error
```

Comment:
there will be an error if `this.el.error` is `null`

Reply:
This PR is going on at https://github.com/clappr/clappr/pull/1614